### PR TITLE
Pricing page updates

### PR DESCRIPTION
**Pull request by lyubomir-popov**
## Done

- UIpdate pricing page as per @lechsandecki 's requests.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes #

## Screenshots

[If relevant, please include a screenshot.]

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
@anthonydillon I think I'm done, can you review please?

File: templates/pricing/pro.html [hunk 1]
```diff
@@ -11,8 +11,8 @@
 
 <section id="ua-infra" class="p-strip--suru-shape-light is-deep">
   <div class="u-fixed-width">
-    <h1 class="p-heading--2 u-no-margin--bottom"><strong>Ubuntu Pro: more security, compliance, and support</strong></h1>
-    <h2 class="u-no-padding--top u-text--muted">Covering all aspects of open infrastructure and applications</h2>
+    <h1 class="p-heading--2 u-no-margin--bottom">Ubuntu Pro: more security, compliance, and support</h1>
```
Comment (anthonydillon) [inline templates/pricing/pro.html:14]:
```suggestion
    <h1 class="p-heading--2 u-no-margin--bottom">Ubuntu Pro: security, compliance, and support</h1>
```

File: templates/pricing/pro.html [hunk 2]
```diff
@@ -26,196 +26,215 @@ <h2 class="u-no-padding--top u-text--muted">Covering all aspects of open infrast
...
-          </tbody>
-        </table>
-      </div>
+    <div class="p-strip is-shallow u-no-padding--top">
+      <h2 class="u-no-margin--bottom">Ubuntu Pro pricing</h2>
+      <p>(including security coverage for Main + Universe)</p>
     </div>
   </div>
-
+  <hr class="is-fixed-width u-no-margin--bottom">
```
Comment (anthonydillon) [inline templates/pricing/pro.html:34]: Should be above the title?

File: templates/pricing/pro.html [hunk 3]
```diff
@@ -26,196 +26,215 @@ <h2 class="u-no-padding--top u-text--muted">Covering all aspects of open infrast
...
+          <tr>
+            <td>Desktop<br><span class="u-text--muted">(workstation/year)</span></td>
+            <td>$25</td>
+            <td>&ndash;</td>
+            <td>$300</td>
+          </tr>
+          <tr>
+            <td>Server with unlimited VMs*<br><span class="u-text--muted">(machine/year)</span></td>
+            <td>$500</td>
+            <td>$17,75</td>
```
Comment (anthonydillon) [inline templates/pricing/pro.html:56]:
```suggestion
            <td>$1,775</td>
```

File: templates/pricing/pro.html [hunk 4]
```diff
@@ -26,196 +26,215 @@ <h2 class="u-no-padding--top u-text--muted">Covering all aspects of open infrast
...
+            <td>$300</td>
```
Comment (anthonydillon) [inline templates/pricing/pro.html:57]:
```suggestion
            <td>$3,400</td>
```

File: templates/pricing/pro.html [hunk 5]
```diff
@@ -26,196 +26,215 @@ <h2 class="u-no-padding--top u-text--muted">Covering all aspects of open infrast
...
+            <td>yes</td>
+          </tr>
+          <tr>
+            <td>Support for Openstack</td>
+            <td>no</td>
+            <td>yes</td>
+            <td>yes</td>
+          </tr>
+          <tr>
+            <td>Support for Ceph Storage (see the life-cycle)</td>
```
Comment (anthonydillon) [inline templates/pricing/pro.html:203]:
```suggestion
            <td>Support for Ceph Storage</td>
```

**Review by anthonydillon — DISMISSED** [review]
Few small suggestions

**Comment by anthonydillon** [comment]
LGTM :+1:

File: templates/pricing/pro.html (hunk 2 above)
Comment (lyubomir-popov) [reply templates/pricing/pro.html]: yes moved it

**Comment by lechsandecki** [comment]
+1

**Comment by lyubomir-popov** [comment]
@lechsandecki happy to approve (add PdM +1 label)? If you reply here I can add the label.

**Comment by lechsandecki** [comment]
> 

yes, +1

**Review by minkyngkm — APPROVED** [review]

**Pull request merged**